pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -1,6 +1,21 @@
-{ stdenv, fetchurl, pkgs, lib }:
+{ mkDerivation, stdenv, fetchurl, pkgs, lib, qtbase }:
+let
 
-stdenv.mkDerivation rec {
+  ldLibraryPath = with pkgs; stdenv.lib.makeLibraryPath  [
+    libbsd.out
+    libffi.out
+    gmpxx.out
+    python27Full.out
+    python27Packages.libxml2.out
```

Comment:
Doesn't just `libxml2` work? Using `stdenv.lib.getLib` is probably more future proof here.

Reply:
No, this is the reason why the derivation broke.

Reply:
I see that it was `python27Packages.libxml2` before, just like it is now. I was suggesting dropping `python27Packages.` prefix and using `stdenv.lib.getLib` function instead of directly asking for `out` output.

Reply:
If you need the bindings, then take python27Packages.libxml2. Indeed, use `getLib`

Reply:
done.